CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
   * `cortex_ruler_client_request_duration_seconds`
 * [ENHANCEMENT] Query-frontend/scheduler: added querier forget delay (`-query-frontend.querier-forget-delay` and `-query-scheduler.querier-forget-delay`) to mitigate the blast radius in the event queriers crash because of a repeatedly sent "query of death" when shuffle-sharding is enabled. #3901
 * [ENHANCEMENT] Ingester: reduce CPU and memory when an high number of errors are returned by the ingester on the write path with the blocks storage. #3969 #3971 #3973
+* [ENHANCEMENT] anLogger: Call `Error` when an `error` is logged
```

Comment:
A couple of things, please:
1. Describe the CHANGELOG entry in a easy to understand way for the final Cortex user
2. Add the PR number at the end

```suggestion
* [ENHANCEMENT] Tracing: flag a span with error if any error log was logged within that span. #3016
```